scripts/build_server_docker

```diff
@@ -4,5 +4,27 @@ set -o errexit
 set -o xtrace
 
 readonly SCRIPTS_DIR="$(dirname "$0")"
+readonly OAK_REMOTE_CACHE_KEY='./.oak_remote_cache_key.json'
+
+echo "TEST"
+# Do we have a json key for the remote cache
+if [ ! -f $OAK_REMOTE_CACHE_KEY ]; then
```

Comment:
Prefer `[[` (see https://google.github.io/styleguide/shell.xml#Test,_%5B_and_%5B%5B ). Also quote variable. I think shellcheck should check some of these things?

Reply:
Done. I don't think shellcheck picked it up. It might miss a rule?